megamek/src/megamek/client/ratgenerator/ForceDescriptor.java

```diff
@@ -854,19 +854,15 @@ private String getForceString() {
             ancestors.add(p);
             p = p.parent;
         }
-        
-        String forceString = "";
+
+        StringBuilder result = new StringBuilder();
         int id = 0;
         for (int i = ancestors.size() - 1; i >= 0; i--) {
             ForceDescriptor ancestor = ancestors.get(i);
             id = 17 * id + ancestor.index + 1;
-            forceString += "\\" + ancestor.parseName() + "|" + id;
-        }
-        // Remove the backslash at the start
-        if (forceString.length() > 0) {
-            forceString = forceString.substring(1);
+            result.append(ancestor.parseName()).append("|").append(id).append("||");
         }
-        return forceString;
+        return result.toString();
```

Comment:
What happened to the backslash?

Reply:
No longer required given the parsing changes. I tested with three layers in MegaMek and MekHQ when working on this PR to ensure it worked.